myfirstproject/templates/product/product-list.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+<h1>Homepage</h1>
+    <ul>
+        {% for object in object_list %}
+            <li><a href="product-list.html" slug=object.slug></a>Product{{ object.product_n}}</li>
```

Comment:
```suggestion
            <li><a href="{ % url "product-list.html" slug=object.slug %}"></a>Product{{ object.product_n}}</li>
```
Таке треба робити з використанням тегів